### DISCUSSION TITLE
FInding "tick_date" start

### DISCUSSION THREAD

**Discussion by alochemes** [bug-or-feature/bc-utils#8] [Q&A]
Hi, first, just wanted to say thank you so very much for bc-utils! It has been an absolute game changer. 

I had a question about how you were able to identify the "tick_date start" parameter for the various contracts in the config file? I wasn't able to find anything on BC's detailed contract information page. 

I am trying to grab a wider array of contracts, and I would be happy to share the list with you when I am finished.

Best,
Andrew

**Comment by bug-or-feature** [comment]
> Hi, first, just wanted to say thank you so very much for bc-utils! It has been an absolute game changer.

Great, I'm glad it is useful :)

> I had a question about how you were able to identify the "tick_date start" parameter for the various contracts in the config file? I wasn't able to find anything on BC's detailed contract information page.

The source was here:
https://www.barchart.com/solutions/data/futures
Tick History column. But I quickly found that it wasn't always accurate. I think subtracted a few months for each exchange. I was a pain to manually edit the individual price files when trying to stitch them together

> I am trying to grab a wider array of contracts, and I would be happy to share the list with you when I am finished.

That would be great, contributions are most welcome. I had planned to expand the list too

**Reply by BorderCollieLover** [reply]
Here is an expanded instrument list in case it helps. There is only "code" and "exchange" information, but I attached a BarChart link for further examination. I hope it's self-explanatory, but let me know if you have any questions. Of course, there might be errors...... 
[ExpandedInstrumentList.xlsx](https://github.com/user-attachments/files/16315698/ExpandedInstrumentList.xlsx)
